openedx_events/bridge/tests/test_utilities.py

```diff
@@ -91,6 +91,16 @@ class TestData:
     sub_test_1: SubTestData1
 
 
+@attr.s(frozen=True)
+class SimpleAttrsWithDefaults:
+    """Test attrs with optional (default=None) values"""
+    boolean_field = attr.ib(type=bool, default=None)
```

Comment:
Quick initial comments:
1. See https://github.com/openedx/openedx-events/issues/38#issuecomment-1021858331 (and other notes in the issue) regarding optional vs default.
2. Is this something we need to deal with now? It may be, and it may be that we need to deal with this sooner rather than later. How will this relate to OpenEdxPublicSignal data validation?

This might warrant its own ticket, and we should think about: a) is this the right time to solve (maybe?), and b) is there a hack that could work for license manager events, especially since we don't even care about this data?

Reply:
We do need to deal with nullable fields, even if we don't call them optional. LicenseEvents have a bunch of things that may be null or may be strings, and right now the bridge can't handle that. I can change the language around the comments and methods to get rid of the idea of optionality but I do think this PR successfully handles nullable fields and so we should push it through if there are no major issues.

Reply:
ADR captures our decision. Marking as resolved.